extensions/route_map/test/mod.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{wrappers, HandlerGroup, HandlerType, Node, RouteMap};
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+use std::collections::{HashMap, HashSet};
+use std::fmt::Debug;
+use std::hash::Hash;
+use std::ptr;
+
+mod init;
+
+fn node_empty(node: &Node) -> bool {
```

Comment:
Maybe move to a `Node` method

Reply:
Even though I literally did it elsewhere in this change, I always forget you can add impl blocks in sub modules. I was gonna say "but it's only used in the tests", but of course.. We can just define it as a method only in the test module.